openhtf/io/user_input.py

```diff
@@ -63,7 +63,8 @@ def __init__(self):
     self._response = None
     self._cond = threading.Condition()
 
-  def DisplayPrompt(self, message, text_input=False):
+  def DisplayPrompt(self, message, text_input=False,
+                    timeout=FLAGS.prompt_timeout_s):
```

Comment:
FLAGS.prompt_timeout_s becomes a primitive (integer). However, it's value is accessed at function declaration, which is at module import, when FLAGS have not been parsed yet. You will either always get the default or whatever gflags does when it's not ready yet.

Also, I'm not sure if this is a wanted feature or not, but FLAGS values can be changed at runtime, so accessing it each time rather than once also enables getting the new value.